src/routes/dev/+page.svelte

```diff
@@ -177,12 +177,12 @@
     </div>
   </QCard>
   <QCard title="Buttons">
-    <QBtn class="q-ma-sm" icon="favorite" label="Using Label" />
+    <QBtn class="q-ma-sm" icon="favorite" label="Using Label" on:activated={console.log} />
```

Comment:
This is intended, since this is on the dev page, right?

Reply:
Yes, it's just to show the on:activated event